### DISCUSSION TITLE
CPU core temperature tray icon font change

### DISCUSSION THREAD

**Discussion by xanda-escuyer** [LibreHardwareMonitor/LibreHardwareMonitor#522] [Q&A]
It seems the status bar/tray icon fonts have changed from a light type to a bold; somewhere between release 0.8.5 & 0.8.8.

This makes it tricky for us to read (especially if using a colour other than black) because the characters have an outline.

There seems to be no way to change this either, not from the UI or within LibreHardwareMonitor.config

![LHWv085](https://user-images.githubusercontent.com/15840995/130615151-9bf1d323-464a-46fb-a4a2-bf808309b3ef.png)
![LHWv088](https://user-images.githubusercontent.com/15840995/130615159-03a2996c-9f0e-4c14-a4ec-135e80b6ca61.png)

If this cannot be configured then may we have the option in a future release please? Thanks.

**Comment by playalisticp** [comment]
See here: https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/issues/931
